src/index.js

```diff
@@ -1,27 +1,37 @@
-const API_URL =  "https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xHcNAgk6UkMi3fEdct8S/scores";
-
-const getData =  ()=>{
-  const response = fetch(API_URL, config);
-  let data = response.json(display);
-  display(data);
-  console.log(data);
-  console.log(data.result[0]);
-
+import './style.css';
+import { postMessage, API_URL } from './modules/submitScore';
+
+const getScores = () => {
+  const displayContent = document.getElementById('table_item');
+  const scoreList = document.createElement('li');
+  const config = {
+    headers: {
+      'Access-Control-Allow-Origin': origin,
+      Accept: 'application/json',
+    },
+  };
+  fetch(API_URL, config)
+    .then((response) => response.json())
+    .then((data) => {
+      for (let i = 0; i < data.result.length; i += 1) {
+        scoreList.textContent = `
+    ${data.result[i].user}
+    ${data.result[i].score}`;
+        displayContent.append(scoreList);
+      }
+    });
```

Comment:
- [ ] <b>REQUIRED:</b> As suggested by the previous reviewer, It would be good to use `async and await` here as it requested by the project requirements. something like...

```js
  const getScores = async () => {
     const { data } = await fetch(API_URL, config)
     // The use data however you want
  }
```
And remember, `await` only work in `async` function.

Reply:
I used this async await method on the file /modules/submnitScore.js. As far as I read, the requirement does not says that the use is mandatory on every function. If you are able to prove me wrong, please do. Meanwhile I will ask for this change to be optional.